smu/geometry/utilities.py

```diff
@@ -0,0 +1,246 @@
+# Utility functions for SMU
```

Comment:
Shoudl be a triple quoted header comment.

Reply:
Yes, done.